docs/registry/README.md

```diff
@@ -0,0 +1,385 @@
+
+```yaml
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Broker
+metadata:
+  name: broker-allow-registered
+spec:
+  ingressPolicy:
+    allowAny: false
+```    
+
+- Auto Add
+
+By setting the ingress policy to auto add, the Broker will accept any event and will add its EventType 
+to the Registry (in case it is not present).
+
+```yaml
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Broker
+metadata:
+  name: broker-auto-add
+spec:
+  ingressPolicy:
+    autoAdd: true 
+```
+
+By not specifying an ingress policy, we thought that the default behavior would be to accept any event, which is similar to `Auto Add` 
+but without registration of events. This needs further discussion.
+
+Note that more policies should probably need to be configured in the future, e.g., allow auto-registration of EventTypes received 
+from within the cluster (e.g., from a response of a Service running in the cluster) as opposed to external services, and so on. 
+We just enumerate two simple cases here.
+
+## FAQ
+
+Here is a list of frequently asked questions that may help clarify the scope of the Registry.
+
+- Is the Registry meant to be used just for creating Triggers or for also setting up Event Sources?
+
+    It's mainly intended for helping the `Event Consumer` with the `discoverability` use case. Therefore, 
+    it is meant for helping out creating Triggers.
+
+- If I have a simple use case where I'm just setting up an Event Source and my KnService is its Sink (i.e., no Triggers involved), 
+is there a need/use for the Registry?
+
+    In this case, we believe there is no need for the Registry. As you can see in the EventType CRD, there is a mandatory 
+    `broker` field. If you are not sending events to a Broker, then there is no need to use a Registry. 
+    Implementation-wise, we can check whether the Source's sink kind is `Broker`, and if so, then register its EventTypes.   
+
+- Is the Registry meant to be used in a single-user environment where the same person is setting up both the Event Source and 
+the destination Sink?
+
+    We believe is mainly intended for multi-user environment. A `Cluster Configurator` persona is in charge of setting up 
+    the Sources, and `Event Consumers` are the ones that create the Triggers. Having said that, it can also be used in 
+    a single-user environment, but the Registry might not add much value compared to what we have right now in terms of 
+    `discoverability`, but it surely does in terms of, for example, `admission control`. 
+    
+- Does a user need to know which type of environment they're in before they should know if they should look at the Registry? 
+In other words, is a Registry always going to be there and if not under what conditions will it be?
+
+    A Registry will always be there, i.e., `Event Consumers` will always be able to `kubectl get eventtypes -n <namespace>`. 
+    In case no CR Sources are pointing to Brokers, then the Registry will be empty. 
```

Comment:
Well, empty registry kind of equals no registry. So, there are certain environments in which a registry (even if present) provides no value. Let's state at the top of the doc what those environments/usecases are. For example, state this the Registry is ONLY for the cases were a Brokers & Triggers are used.

Reply:
I though it was clear from the whole context, and some of the other answers/discussions. 
It seems it's not, so let me explicitly call that out. Thanks for bringing that up. Will do